server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/HealthCheckRoutes.java

```diff
@@ -65,27 +74,49 @@ public String getBasePath() {
 
     @Override
     public void define(Service service) {
-        this.service = service;
-
-        validateHealthchecks();
+        service.get(HEALTHCHECK, this::validateHealthchecks, jsonTransformer);
+        service.get(HEALTHCHECK + "/checks/:componentName", this::performHealthCheckForComponent, jsonTransformer);
```

Comment:
could you extract `"componentName"` into a constant, and reuse that constant in `request.params("componentName")` of `performHealthCheckForComponent()` ?

Reply:
fixed in e1464f0e927b650a861518b8f9df17e874419e72